ParkingLot/ParkingLot.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.ConstrainedExecution;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ParkingLotns
+{
+    public class ParkingLot
+    {
+        private int capacity = 10;
+        private Dictionary<string, string> ticket2Car = new Dictionary<string, string>();
+        private List<string> usedTickets = new List<string>();
+
+        public string Fetch(string ticket)
+        {
+            if (ticket2Car.ContainsKey(ticket))
+            {
+                string foundCar = ticket2Car[ticket];
+                usedTickets.Add(ticket);
```

Comment:
WangKe:
usedTickets data structure works, and we could also remove the ticket from ticket2Car in order to accomplish the unused-ticket function